common/util.go

```diff
@@ -434,14 +433,16 @@ func CreateHistoryStartWorkflowRequest(
 		StartRequest: startRequest,
 	}
 
-	firstDecisionTaskBackoffSeconds := backoff.GetBackoffForNextScheduleInSeconds(
-		startRequest.GetCronSchedule(), now, now)
 	delayStartSeconds := startRequest.GetDelayStartSeconds()
-	if delayStartSeconds > 0 && firstDecisionTaskBackoffSeconds > 0 {
-		return nil, ErrDelayStartSeconds
```

Comment:
We may update the function signature to only return the request. Error is not needed?

Reply:
With this PR, user can set both delayStart and CronSchedule. Previously they were treated as conflicting params. When do you think we should return an error?

Reply:
Yeah. I think after it supports both delayStart and CronSchedule. It doesn't need to return an error. Feel free to do it in this PR or in a different PR.

Reply:
Oh I see, what you mean now, I will update the signature